modin/pandas/test/dataframe/test_iter.py

```diff
@@ -289,6 +290,34 @@ def test___setattr__mutating_column():
     df_equals(modin_df, pandas_df)
     assert modin_df.col0.equals(modin_df["col0"])
 
+    # Check that attempting to add a new col via attributes raises warning
+    # and adds the provided list as a new attribute and not a column.
+    with pytest.warns(
+        UserWarning,
+        match="Modin doesn't allow columns to be created via a new attribute name - see",
```

Comment:
This warning should be probably extended. "see ..." what?

Reply:
Done.